app/grandchallenge/core/static/js/markdownx_full_page.js

```diff
@@ -0,0 +1,18 @@
+// Sync the height of the preview element to the height of the editor element.
+// This lets the user adjust the height of both elements by resizing the editor.
+$(document).ready(() => {
+    const ELEMENTS = document.getElementsByClassName("markdownx");
+    Object.values(ELEMENTS).map(element => {
```

Comment:
No need for all caps here.

```suggestion
    const elements = document.getElementsByClassName("markdownx");
    Object.values(elements).map(element => {
```